frontend/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
```

Comment:
React 18 is too new and other frameworks may not still be supporting the new React version. Lets use React 17 which is he last stable version of React

Reply:
sure I will make the changes